clients/src/main/resources/common/message/ApiVersionsResponse.json

```diff
@@ -47,7 +47,7 @@
       "about": "The duration in milliseconds for which the request was throttled due to a quota violation, or zero if the request did not violate any quota." },
     { "name":  "SupportedFeatures", "type": "[]SupportedFeatureKey", "ignorable": true,
       "versions":  "3+", "tag": 0, "taggedVersions": "3+",
-      "about": "Features supported by the broker. Note: in v0-v3, features with MinSupportedVersion = 0 show up with MinSupportedVersion = 1.",
+      "about": "Features supported by the broker. Note: in v0-v3, features blocked with MinSupportedVersion = 0.",
```

Comment:
Pardon me, I don't catch "features blocked with MinSupportedVersion = 0."

Also,  please fix `BrokerRegistrationRequest.json` too

Reply:
> Pardon me, I don't catch "features blocked with MinSupportedVersion = 0."
I change it to omit, it is more accurancy in this case